### PR TITLE
Document freeze guards

### PR DESCRIPTION
**Pull request by lorisleiva**
- Write "[Freeze Sol Payment](https://docs-git-loris-freeze-guards-metaplex.vercel.app/programs/candy-machine/available-guards/freeze-sol-payment)" guard documentation.
- Write "[Freeze Token Payment](https://docs-git-loris-freeze-guards-metaplex.vercel.app/programs/candy-machine/available-guards/freeze-token-payment)" guard documentation.
- Order guard documentation alphabetically.

### CODE REVIEW

File: docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+In the example below, we initialize the Freeze Escrow account with a maximum Freeze Period of 15 days and we use the current identity as the Candy Guard authority.
+
+```tsx
+await metaplex.candyMachines().callGuardRoute({
+  candyMachine,
+  guard: 'freezeSolPayment',
+  settings: {
+    path: 'initialize',
+    period: 15 * 24 * 60 * 60, // 15 days.
```
Comment (febo) [inline docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md:103]: Would it make sense to add an utility function to convert days/hours to seconds? E.g., `secondsFromDays`, `secondsFromHours`.

File: docs/01-programs/02-candy-machine/09-available-guards/06-freeze-token-payment.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+- The Candy Machine was deleted.
+- The configured Freeze Period — which can be a maximum of 30 days — has passed.
+
+The tokens are transferred to a "Freeze Escrow" account which must be initialized by the Candy Guard authority before minting can start. Once all Frozen NFTs have been thawed, the funds can be unlocked and transferred to the configured destination account by the Candy Guard authority.
+
+You may initialize the Freeze Escrow account, thaw NFTs and unlock funds [via the route instruction](#route-instruction) of this guard.
+
+![CandyMachinesV3-GuardsFreezeTokenPayment2.png](/assets/candy-machine-v3/CandyMachinesV3-GuardsFreezeTokenPayment2.png#radius)
+
+## Guard Settings
```
Comment (febo) [inline docs/01-programs/02-candy-machine/09-available-guards/06-freeze-token-payment.md:24]: Same overview diagram as for `freeze sol payment`?

File: docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md (hunk 1 above)
Comment (lorisleiva) [reply docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md:103]: I wanted to do that on the SDK initially but thought it's maybe too generic and people would already have their way of translating time within their framework. I'm happy to revisit this later though.

File: docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+The **Freeze Sol Payment** guard allows minting frozen NFTs by charging the payer an amount in SOL. Frozen NFTs cannot be transferred or listed on any marketplaces until thawed.
+
+Frozen NFTs can be thawed by anyone as long as one of the following conditions is met:
+- The Candy Machine has minted out.
+- The Candy Machine was deleted.
+- The configured Freeze Period — which can be a maximum of 30 days — has passed.
+
+The funds are transferred to a "Freeze Escrow" account which must be initialized by the Candy Guard authority before minting can start. Once all Frozen NFTs have been thawed, the funds can be unlocked and transferred to the configured destination account by the Candy Guard authority.
+
+You may initialize the Freeze Escrow account, thaw NFTs and unlock funds [via the route instruction](#route-instruction) of this guard.
```
[the remark this answers is not in the document]
Comment (lorisleiva) [reply docs/01-programs/02-candy-machine/09-available-guards/05-freeze-sol-payment.md:20]: Great idea, working on it!

**Comment by febo** [comment]
Looks great!

**Review by febo — APPROVED** [review]

**Pull request merged**